### DISCUSSION TITLE
Streetnames

### DISCUSSION THREAD

**Discussion by Lamandus** [cataclysmbn/Cataclysm-BN#4941] [General]
### Describe the solution you'd like

We have cities, but we don't have any street names. It would help the immersion if we can add street names to the cities. House numbers are secondary. Especially in bigger cities you quickly get lost, since you don't know exactly where you are. Having a way to determine which street you are on, would help quite a bit.
1. Naming should follow a easy solution, for example, first name streets going from north to south, then east to west.
2. The street name should be visible in the map view, for better display in the description, alternatively on the road itself (but that would compromise visibility)
3. The house numbers should also follow an easy N->S, E->W pattern. Cornerhouses should get the number from where the building is pointing to.
4. We use the "names" json to expand on street names.

**Comment by Ignaramico** [comment]
what exactly do you mean by getting lost? the map (defaul key m) is pretty accurate in regard to street, and even more so with the auto walk/drive.
the only way to get lost would be in a map special (like a lab, since they have a lot of walls/doors on a same overmap tile)

**Comment by Lamandus** [comment]
> what exactly do you mean by getting lost? the map (defaul key m) is pretty accurate in regard to street, and even more so with the auto walk/drive. the only way to get lost would be in a map special (like a lab, since they have a lot of walls/doors on a same overmap tile)

Getting lost as in, what you already have seen and what not (in the map view). But mostly the addition would mean a enhancement in making the game world better (forgot the word)... Immersion! That was the word

**Comment by chaosvolt** [comment]
This would basically be just a flavor thing to be honest, and this feels like it wasn't really worth being an issue report when it could've been a discussion.

**Comment by Lamandus** [comment]
But the issue panel also asks for feature requests...

**Comment by Lamandus** [comment]
![Screenshot_20240706_135405_Opera](https://github.com/cataclysmbnteam/Cataclysm-BN/assets/33199510/91b98bf1-fe75-4d2a-9cb7-bf4732834eaa)

**Comment by chaosvolt** [comment]
We really don't need to clog up issues with minor things like this though. Especially since this sorta thing has no actual in-game benefit and would be purely for flavor.
